### DISCUSSION TITLE
Transfer mittels Transaktion

### DISCUSSION THREAD

**Discussion by trobanga** [medizininformatik-initiative/fts-next#430] [Ideas]
Transfer mittels Transaktion (inkl. Rollback, falls Fehler aufgetreten sind)

**Comment by SebStaeubert** [comment]
Dies ist eine Anforderung, die am 28.11.2024 in der MII Taskforce übergreifende Schnittstellen vom Standort Aachen geäußert wurde.

**Comment by SebStaeubert** [comment]
Ziel:
- inkonsistente Zustände verhindern, z.B. unvollständige Übertragungen 

Maßnahmen:
- Reports
  - Liste der vollständig übertragenen Patienten
  - Liste der unvollständig übertragenen Patienten
- Erneute Übertragung der unvollständig übertragenen Patienten
- Automatisierung

Achtung:
- Rollback ist nach derzeitiger Einschätzung nur sehr aufwendig zu implementieren
  - ggf. umsetzen, wenn Funktionen zum Widerruf implementiert werden
